layers/meta-opentrons/recipes-robot/python3-jupyter/files/jupyter-notebook.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Jupyter notebook server
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+ExecStart=/bin/sh -c "jupyter notebook --allow-root --config=%E/jupyter/jupyter_notebook_config.py --notebook-dir=%S/jupyter/notebooks 2>&1"
+PIDFile=%t/jupyter/jupyter.pid
+SupplementaryGroups=dialout
+StateDirectory=jupyter jupyter/notebooks jupyter/data jupyter/config ipython
+RuntimeDirectory=jupyter
+Environment=JUPYTER_RUNTIME_DIR=%t/jupyter JUPYTER_CONFIG_DIR=%S/jupyter/config JUPYTER_DATA_DIR=%S/jupyter/data IPYTHONDIR=%S/ipython JUPYTER_NOTEBOOK_DIR=%S/jupyter/notebooks OT_SMOOTHIE_ID=AMA RUNNING_ON_PI=true
```

Comment:
we need to update these to be the ot-3 specific ones (see the environment flags set in the robot-server unit) rather than the ot-2 specific ones